src/components/avatar/avatar.spec.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
We want to avoid adding more `enzyme` tests to `ui-kit` Can you rewrite those to `react-testing-library`?

You can drop the tests which are concerned with visual stuff (like dom snapshots or class assertions) as we will move to Visual Regression Testing for these things.